packages/graphql/package.json

```diff
@@ -31,7 +31,7 @@
     "subscriptions-transport-ws": "0.11.0",
```

Comment:
This package still has the vulnerable `ws` package installed. Like savilde said here [https://github.com/nestjs/graphql/issues/3215](url) this package is deprecated and should be changed to `graphql-ws`

Reply:
refs https://github.com/nestjs/graphql/issues/2775